### ISSUE TITLE
Can't deploy any RedMOD with scripts (WolvenKit doesn't read metadata.json)

### ISSUE THREAD

**Issue by Khotul** [WolvenKit/WolvenKit#1005]
Used the search bar but didn't find another Issue describing this. 
Found this when working on my mod (ExtendedPerks), took me over an hour to find what caused the issue.

**Deploying with WolvenKit:**

```
2022-11-10 00:41:13.327 [INF] WorkDir: E:\SteamLibrary\steamapps\common\Cyberpunk 2077\tools\redmod\bin\redMod.exe
2022-11-10 00:41:13.327 [INF] Running commandlet: deploy -root="E:\SteamLibrary\steamapps\common\Cyberpunk 2077" -mod="ExtendedPerks" "Mod_LightweightDifficultyEnhancer"
2022-11-10 00:41:13.829 [INF] Copyright (c) 2022 CD Projekt Red. All Rights Reserved.
2022-11-10 00:41:13.829 [INF] Created in collaboration with CD Projekt Vancouver and Yigsoft LLC
2022-11-10 00:41:13.829 [INF] 
2022-11-10 00:41:13.829 [INF] Continuing without global backend resource depot.
2022-11-10 00:41:13.829 [INF] Core: App instance initialized in 0.00 ms
2022-11-10 00:41:13.829 [INF] 
2022-11-10 00:41:13.829 [INF] 
2022-11-10 00:41:13.829 [INF] [DEPLOY] Stage 1/5 - Initialization
2022-11-10 00:41:13.830 [INF] Found mod "Extended Perks" (v0.1.3) in folder "ExtendedPerks" (enabled; not deployed; TWEAKS; SCRIPTS; )
2022-11-10 00:41:13.830 [INF] Found mod "Lightweight Difficulty Enhancer" (v1.02) in folder "Mod_LightweightDifficultyEnhancer" (enabled; not deployed; TWEAKS; )
2022-11-10 00:41:13.830 [INF] Needs deployment: true
2022-11-10 00:41:13.830 [INF] [DEPLOY] Stage 2/5 - Script Compilation
2022-11-10 00:41:18.247 [ERR] ScriptCompiler: ERROR: Script binding error: Missing native class 'ActionInternalEvent'
2022-11-10 00:41:19.402 [ERR] ERROR: Script compilation has failed!
2022-11-10 00:41:19.512 [INF] ScriptCompiler: Failed to bind compiled scripts
2022-11-10 00:41:19.512 [INF] 
2022-11-10 00:41:19.512 [INF] Commandlet deploy has failed.
2022-11-10 00:41:19.705 [INF] Exit time    : 11/10/2022 12:41:19 AM
Exit code    : 8
Elapsed time : 6227
```

**Deploying with redmod.exe** using the following command `redmod deploy -mod="ExtendedPerks" -reportProgress -rttiSchemaPath="E:\SteamLibrary\steamapps\common\Cyberpunk 2077\tools\redmod\metadata.json"`

 
```
Copyright (c) 2022 CD Projekt Red. All Rights Reserved.
Created in collaboration with CD Projekt Vancouver and Yigsoft LLC

No root specified, using default root path.
Continuing without global backend resource depot.
Core: App instance initialized in 0.00 ms


[DEPLOY] Stage 1/10 - Initialization
Found mod "Extended Perks" (v0.1.3) in folder "ExtendedPerks" (enabled; not deployed; TWEAKS; SCRIPTS; )
Found mod "Lightweight Difficulty Enhancer" (v1.02) in folder "Mod_LightweightDifficultyEnhancer" (disabled; not deployed; TWEAKS; )
Needs deployment: true
[DEPLOY] Stage 2/10 - Script Compilation
File 'E:\SteamLibrary\steamapps\common\Cyberpunk 2077\r6\cache\modded\final.redscripts' (12649652 bytes) written.
[DEPLOY] Stage 3/10 - TweakDB Compilation
[DEPLOY] Stage 4/10 - TweakDB Parsing
[DEPLOY] Stage 5/10 - TweakDB Processing
[DEPLOY] Stage 6/10 - TweakDB Flattening
[DEPLOY] Stage 7/10 - TweakDB Processing queries
[DEPLOY] Stage 8/10 - TweakDB Serializing
File 'E:\SteamLibrary\steamapps\common\Cyberpunk 2077\r6\cache\modded\tweakdb.str' (21738651 bytes) written.
[DEPLOY] Stage 9/10 - Audio
[DEPLOY] Stage 10/10 - Finalize
File 'E:\SteamLibrary\steamapps\common\Cyberpunk 2077\r6\cache\modded\mods.json' (298 bytes) written.

Commandlet deploy has succeeded. 
````

<sub>(I know WolvenKit compiles also a second mod and the command I used with redmod doesn't, however the second mod doesn't have any scripts so it's not relevant to this issue at all.)</sub>

**Comment by rfuzzo** [comment]
what redmod version do you have? (You can check by right-clicking on redmod.exe and view the details.) It should say "1.61"

**Comment by rfuzzo** [comment]
also which wkit version are you on? 8.7 does not support game version 1.61

**Comment by Khotul** [comment]
Due to slow internet connection, I prevented Cyberpunk auto updates ([using this guide](https://www.reddit.com/r/Steam/comments/5zxn9d/how_to_skip_updates_for_appsgames_installed_via/)) and I'm still on game version 1.6
Redmod version 1.6
WolvenKit version 8.7.0

I could update my Cyberpunk and WolvenKit to try if the issue still occurs on the new version, but the new update is over 20GB so the soonest I would get back to you is in about two days.

Also checked the `wolvenkit_plugins.json` file and it displays redmod version as null (and doesn't list any files).
```
  "redmod": {
    "Id": 4,
    "Version": null,
    "Files": []
  }, 
```
Not sure if that has anything to do with this issue, added the path and version here manually but it didn't change anything.

**Comment by rfuzzo** [comment]
> Also checked the wolvenkit_plugins.json file and it displays redmod version as null (and doesn't list any files).
That's normal.

and, hmm, that might be a bug in the 8.7 release of WolvenKit. However, it should be fixed in the nightly release (which is for 1.61)

**Comment by rfuzzo** [comment]
cannot reproduce this. redmod deploy works on main
